BuildingSync.xsd

```diff
@@ -17787,4 +17790,468 @@
+                                </xs:annotation>
+                              </xs:element>
+                              <xs:element name="Beta2" type="xs:decimal" minOccurs="0" maxOccurs="1">
+                                <xs:annotation>
+                                  <xs:documentation>In both three parameter models, this is the change point.  In the 4p and 5p models, this is beta_cdd.</xs:documentation>
+                                </xs:annotation>
+                              </xs:element>
+                              <xs:element name="Beta3" type="xs:decimal" minOccurs="0" maxOccurs="1">
+                                <xs:annotation>
+                                  <xs:documentation>In the 4p models, this is the change point (as there is only one change point).  In the 5p models, this is the lower value change point, which in CalTRACK terms is referred to as the heating change point.</xs:documentation>
+                                </xs:annotation>
+                              </xs:element>
+                              <xs:element name="Beta4" type="xs:decimal" minOccurs="0" maxOccurs="1">
+                                <xs:annotation>
+                                  <xs:documentation>In the 5p model, this is the upper value change point, which in CalTRACK terms is referred to as the cooling change point.</xs:documentation>
+                                </xs:annotation>
+                              </xs:element>
+                            </xs:sequence>
+                          </xs:complexType>
+                        </xs:element>
+                        <xs:element name="TimeOfWeekTemperatureModel">
+                          <xs:complexType>
+                            <xs:sequence>
+                              <xs:element name="DUMMY"/>
+                            </xs:sequence>
+                          </xs:complexType>
+                        </xs:element>
+                      </xs:choice>
+                    </xs:complexType>
+                  </xs:element>
+                  <xs:element name="DerivedModelPerformance">
+                    <xs:annotation>
+                      <xs:documentation>Characterization of the performance of the model.</xs:documentation>
+                    </xs:annotation>
+                    <xs:complexType>
+                      <xs:sequence>
+                        <xs:element name="RSquared" minOccurs="0" maxOccurs="1">
+                          <xs:simpleType>
+                            <xs:restriction base="xs:decimal">
+                              <xs:minInclusive value="0"/>
+                              <xs:maxInclusive value="1"/>
+                            </xs:restriction>
+                          </xs:simpleType>
+                        </xs:element>
+                        <xs:element name="AdjustedRSquared" minOccurs="0" maxOccurs="1">
```

Comment:
Annotation?

Reply:
Adjusted R-Squared is typically used to compare model fits across models generated with different numbers of parameters, since R-Squared is unable to account for model complexity (i.e. quadratic, cubic, etc.).  It uses a penalty for the number of terms in a model, and can be calculated as: Adj-R-Squared = 1 - (SS_resid / SS_total) * ((n - 1) / (n - d - 1)). Here, SS_resid is the sum of the squared residuals from the regression, and SS_total is the sum of the squared differences from the mean of the dependent variable (total sum of squares).  n is the number of observations in the data, and d is the degree of the polynomial.  See: https://www.mathworks.com/help/matlab/data_analysis/linear-regression.html#f1-15010.

Reply:
fixed